packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -483,7 +483,13 @@ class WebServiceWorker extends Target {
         file.path.endsWith('.part.js.map')) {
         continue;
       }
-      final String url = globals.fs.path.toUri(
+
+      // because caching logic expects RESOURCES without leading slash
+      final String baseHrefWithoutLeadingSlash = environment.defines[kBaseHref] != null
+        ? environment.defines[kBaseHref]!.substring(1)
```

Comment:
Quick question about this. Is there any validation that the base href always starts with a single `/`?

Can base href be: `//stuff` (I think this would work just fine, it'd ignore the double slashes and assume only one) or `stuff/` (no slash at all?)?

Reply:
Thanks for checking @davidlanderos-digital !